microsoft-edge/web-platform/release-notes/136.md

```diff
@@ -0,0 +1,500 @@
+
+
+<!-- ---------- -->
+###### Deprecate `Intl.Locale` getters
+
+The `Intl.Locale` API exposes information for a locale, including its week elements and its hour cycle type, such as 12-hour clock, Japanese 12-hour clock, or 24-hour clock.
+
+The accessor properties of the `Intl.Locale` object are now deprecated, in favor of the corresponding functions, per the specification.  For example, the `Intl.Locale.prototype.hourCycle` accessor property is deprecated in favor of the `Intl.Locale.prototype.getHourCycles()` function.
+
+See [Intl.Locale](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale) at MDN.
+
+
+<!-- ====================================================================== -->
+## Origin trials
+
+The following are new experimental APIs that you can try on your own live website for a limited time.
+
+To learn more about origin trials, see [Use origin trials in Microsoft Edge](../../origin-trials/index.md).
+
+To see the full list of available origin trials, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials/).
+
+
+<!-- ------------------------------ -->
+#### New Chromium origin trials
+
+
+<!-- ---------- -->
+###### Update of Canvas text rendering implementation
+
+Expires on December 16, 2025.
+
+New implementation of `measureText()`, `fillText()`, and `strokeText()` for the CanvasRenderingContext2D API.  You can compare the performance of these methods against the previous implementation of these methods.
+
+* [Explainer](https://docs.google.com/document/d/1UbHuoflNyLbSavpH86HEmXFGyz2V4uZ6cjNpuuwrtBg/edit?usp=sharing)
+* [Feedback](https://crbug.com/new)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/3227392082964381697)
+
+
+<!-- ---------- -->
+###### Set preferred audio output device
+
+Expires on January 27, 2026.
+
+This feature allows a page to set a non-default device for all audio output from the page, including its child frames.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/SetPreferredSinkId/explainer.md)
+* [Feedback](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/1173750652883435521)
+
+
+<!-- ---------- -->
+###### `confidence` field on `PerformanceNavigationTiming` object
+
+Expires on October 14, 2025.
+
+The `PerformanceNavigationTiming` object now includes a `confidence` field, which allows you to assess whether the navigation timings are representative of your web app's performance.  
+
+You add noise to this field via a randomized response algorithm, so that an aggregator can debias the data, but individual records cannot used to track users.  Due to this noise, the `confidence` field isn't immediately useful.  By collecting this field in your Real User Monitoring (RUM) data, after collecting enough records, you can analyze your data and eliminate the noise.
```

Comment:
```suggestion
The `PerformanceNavigationTiming` object now includes a new confidence field, allowing developers to assess whether the navigation timings are representative of their web app's performance. Navigation Timing is extremely useful for understanding the performance of their site. However, operating system and browser activity can cause this data to be noisy. By collecting this field in their Real User Monitoring (RUM) data, with enough records, developers can analyze their data and eliminate the noise.
```

This isn't quite right, how about this?

Reply:
I think we should say this:

The PerformanceNavigationTiming object now includes a new confidence field, enabling developers to evaluate whether navigation timings accurately reflect their web app's performance. However, because this field employs differential privacy to safeguard user privacy, the data initially contains some noise. By gathering this field in their Real User Monitoring (RUM) data, developers can accumulate enough records to analyze trends and filter out the noise, gaining meaningful insights over time.

Reply:
Prefer "you" over "the developer".
* Per corp style https://learn.microsoft.com/style-guide/grammar/person#in-general-use-second-person-pronouns-you-your
* For clarity through leveraging consistency in this doc-set, 

The audience for LMC, including this docset, is you, the developer.

@captainbrosset @aluhrs13  @mwjacksonmsft 

I am incorp'g @mwjacksonmsft draft, which modifies @aluhrs13 draft, and applying a writer copyedit.

Reply:
```
Expires on October 14, 2025.

The `PerformanceNavigationTiming` object now includes a `confidence` field.  Use the `confidence` field to evaluate whether navigation timings accurately reflect your web app's performance.  The data initially contains some noise, because this field employs differential privacy, to safeguard user privacy.  By gathering this field in your Real User Monitoring (RUM) data, you can accumulate enough records to analyze trends and filter out the noise, gaining meaningful insights over time.
```